### PR TITLE
Bump to version 2.9.0

### PR DESCRIPTION
**Pull request by Mehdi-Bendriss**

### CODE REVIEW

**Review by welpaolo — APPROVED** [review]
LGTM

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -91,6 +91,12 @@ jobs:
               oci-archive:charmed-opensearch_${version}_amd64.rock \
               docker-daemon:charmed-opensearch:${version}
 
+      - name: Setup the required system configs
+        run: |
+          sudo sysctl -w vm.swappiness=0
+          sudo sysctl -w vm.max_map_count=262144
+          sudo sysctl -w net.ipv4.tcp_retries2=5
```
Comment (zmraul) [inline .github/workflows/ci.yaml:98]: How is this useful on a rock?

Comment (Mehdi-Bendriss) [reply .github/workflows/ci.yaml:98]: Not on the rock itself, but to configure the host for the integration tests which essentially create a 3 nodes cluster (3 docker containers) and test the cluster formation

**Review by paulomach — APPROVED** [review]
good effort

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (zmraul) [reply .github/workflows/ci.yaml:98]: Sorry, didn't notice this was on the host for CI. Thanks for clearing that :)

**Review by zmraul — APPROVED** [review]

**Pull request merged**